code/modules/projectiles/updated_projectiles/guns/specialist.dm

```diff
@@ -17,7 +17,7 @@
 	icon_state = "m42a"
 	item_state = "m42a"
 	max_shells = 15 //codex
-	caliber = "10x28mm Caseless" //codex
+	caliber = "10x28mm" //codex
```

Comment:
This is not a proper fix as other guns are likely to have the same issues, a codex variable shouldn't affect regular gameplay.

Reply:
this isnt a codex variable the comment is wrong

Reply:
Then the comment should be removed. Either way is fine, but right now the PR is not okay.